test/unit/error_events.test.js

```diff
@@ -70,13 +69,6 @@ test('Error events', (t) => {
         const exception = new Exception({error, customAttributes, timestamp})
         tx.addException(exception)
 
-        const segment = tx.agent.tracer.getSegment()
-
-        const spanAttributes = segment.attributes.get(DESTINATIONS.SPAN_EVENT)
-
-        t.equal(spanAttributes['error.class'], 'Error')
-        t.equal(spanAttributes['error.message'], 'some error')
-
```

Comment:
These made no sense to test here and would have required some serious replumbing of the test.

Reply:
👍